From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: deepak1556 <hop2deep@gmail.com>
Date: Thu, 20 Sep 2018 17:50:13 -0700
Subject: chrome_key_systems.patch

Disable persiste licence support check for widevine cdm,
as its not supported in the current version of chrome.

diff --git a/chrome/renderer/media/chrome_key_systems.cc b/chrome/renderer/media/chrome_key_systems.cc
index 13fea00df7ea693ff10576be264f17eafe8f80bf..ee5966ee7e0503d9dd43b407e605952a73aff309 100644
--- a/chrome/renderer/media/chrome_key_systems.cc
+++ b/chrome/renderer/media/chrome_key_systems.cc
@@ -17,7 +17,9 @@
 #include "base/strings/utf_string_conversions.h"
 #include "build/build_config.h"
 #include "build/chromeos_buildflags.h"
+#if 0
 #include "chrome/renderer/chrome_render_thread_observer.h"
+#endif
 #include "components/cdm/renderer/external_clear_key_key_system_info.h"
 #include "components/cdm/renderer/widevine_key_system_info.h"
 #include "content/public/renderer/render_thread.h"
@@ -238,12 +240,14 @@ SupportedCodecs GetSupportedCodecs(const media::CdmCapability& capability,
 
 // Returns whether persistent-license session can be supported.
 bool CanSupportPersistentLicense() {
+#if 0
   // Do not support persistent-license if the process cannot persist data.
   // TODO(crbug.com/457487): Have a better plan on this. See bug for details.
   if (ChromeRenderThreadObserver::is_incognito_process()) {
     DVLOG(2) << __func__ << ": Not supported in incognito process.";
     return false;
   }
+#endif
 
 // On ChromeOS, platform verification is similar to CDM host verification.
 #if BUILDFLAG(ENABLE_CDM_HOST_VERIFICATION) || BUILDFLAG(IS_CHROMEOS)
